### PR TITLE
Ts and rollup setup

### PR DESCRIPTION
**Pull request by perarnborg**
#2 #9 

First stab to setup typescript and build with Rollup.

Also added very basic dev env docs to README and docs.

### CODE REVIEW

**Review by dennerkrans — APPROVED** [review]
Nice

**Comment by perarnborg** [comment]
@dennerkrans This one needs another approval after resolving merge conflicts

**Review by dennerkrans — APPROVED** [review]

**Pull request merged**
